src/Storages/ObjectStorageQueue/StorageObjectStorageQueue.cpp

```diff
@@ -953,6 +1021,14 @@ static const std::unordered_set<std::string_view> changeable_settings_ordered_mo
     "cleanup_interval_min_ms",
     "use_persistent_processing_nodes",
     "persistent_processing_node_ttl_seconds",
+    "after_processing_move_uri",
```

Comment:
If this is marked as changeable (via ALTER), then need to also adjust this code https://github.com/ClickHouse/ClickHouse/blob/cb455811e163ae576a1ec629894f5d26dee5544d/src/Storages/ObjectStorageQueue/StorageObjectStorageQueue.cpp#L1267-L1271